lsp/server/src/server.ts

```diff
@@ -155,6 +140,54 @@ connection.languages.diagnostics.on(async (params) => {
     }
 });
 
+connection.onCodeAction((params) => {
+    const textDocument = documentCache.get(params.textDocument.uri);
+    const diagnostics = params.context.diagnostics;
+	if (textDocument === undefined || diagnostics.length === 0) {
+	    return undefined;
+	}
+    
+	const result: CodeAction[] = [];
+
+    diagnostics.forEach((diagnostic) => {
+        // generate the two suppressing quick fixes 
+        const { data : producerId } = diagnostic;
+        if (typeof producerId === 'string') {
```

Comment:
Do you need to check the type of `producerId`?

Reply:
you are right, we don't need it for now, let me remove it

Reply:
done